app/controllers/api/v1/helpers/lumpsum.rb

```diff
@@ -0,0 +1,23 @@
+module API
+  module V1
+    module Helpers
+      module Lumpsum
+        extend Grape::API::Helpers
+
+        def display_all_lumpsums
+          ::Lumpsum.all
+        end
+
+        def create_new_lumpsum(params)
+          invested_amount = params[:total_investment]
+          total_value = invested_amount * ((1 + (params[:expected_return_rate] / (100 * 1.0))) ** params[:time_period])
+          estimated_returns = total_value - invested_amount
+          date_of_application = Date.today
```

Comment:
use `Time.zone.now` instead.
Rename method to `create_new_lumpsum` to `create_lumpsum`.

Since you are in a controller of lumpsum, so create a lumpsum and associate user with it instead.